tinder.js

```diff
@@ -67,15 +67,21 @@ function TinderClient() {
    * @param {Function} callback the callback to invoke when the request completes
    * @param {Object} form the form to send if we need multipart format
    */
-  var tinderPost = function(path, data, callback, form) {
-    var opts = getRequestOptions(path, data, form);
+  var tinderPost = function(path, data, callback) {
+    var opts = getRequestOptions(path, data);
+    opts.method = 'POST';
+    request(opts, callback);
+  };
+
+  var tinderMultipartPost = function(path, data, callback) {
+    var opts = getRequestOptions(path, null);
+    opts['url'] = TINDER_IMAGE_HOST + path,
     opts.method = 'POST';
     var req = request(opts, callback);
-    if(form){
-      var fm = req.form();
-      for (var key in form) {
-        fm.append(key,form[key]);
-      }
+    var fm = req.form();
```

Comment:
I have to use the `.form` method to send in multipart mode, I can't do it with `opts['formData']` directly , even if in the request documentation it's seems that we can...